packages/core/rollup.config.js

```diff
@@ -1,20 +1,21 @@
 import buble from 'rollup-plugin-buble'
-import nodeResolve from 'rollup-plugin-node-resolve'
 
 export default {
   entry: 'src/index.js',
   plugins: [
-    buble(),
-    nodeResolve({
-      jsnext: true
-    })
+    buble()
   ],
   targets: [
     {
       dest: 'dist/mostCore.js',
       format: 'umd',
       moduleName: 'mostCore',
-      sourceMap: true
+      sourceMap: true,
+      globals: {
```

Comment:
u should also put:
`external: ['@most/scheduler', '@most/disposable', '@most/prelude'],`
at the top level of the `rollup.config.js` to prevent nasty console.warns

Reply:
Thanks @Andarist.  Added external to relevant rollup configs in f64674a.